substrate/frame/revive/rpc/src/tests.rs

```diff
@@ -32,9 +32,9 @@ use static_init::dynamic;
 use std::thread;
 use substrate_cli_test_utils::*;
 
-/// Create a websocket client with a 30s timeout.
+/// Create a websocket client with a 120s timeout.
```

Comment:
So the node takes more than 30s to serve the rpc endpoint? I thought the issue was with the overall test taking more than 30s

Reply:
Nope. It was failing here. Thenextest allots 2x60s before it kills you.